### DISCUSSION TITLE
Sphinx docs convention for the ruff linter

### DISCUSSION THREAD

**Discussion by IrishWhiskey** [astral-sh/ruff#11159] [Q&A] [answered]
The `tool.ruff.lint.pydocstyle` config supports `google`, `numpy` and `pep257`. Is there any plan to add the [Sphinx style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html)? It looks just as popular as the others to me.

**Comment by dhruvmanila** [comment, marked as answer]
Related https://github.com/astral-sh/ruff/issues/6606

**Reply by IrishWhiskey** [reply]
Thank you Dhruv!
